.github/workflows/main.yml

```diff
@@ -85,6 +86,22 @@ jobs:
           SLACK_USERNAME: RedKubesBot
         if: always() # failure
 
+  integration:
+    runs-on: self-hosted
+    needs: build-test-push-release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: K8s in Docker
+        run: |
+          docker login -u otomi -p $DOCKER_PASSWORD
+          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock --network kind otomi/kind:latest
+          docker cp kind-control-plane:/etc/kubernetes/admin.conf $PWD/kind/kubeconfig
+          chmod +r ./kind/kubeconfig
```

Comment:
can we make only owner to read this file
```
chmod 400
```

Reply:
I don't think it is convenient, the app user is then not allowed to read.

Reply:
sure, it is mounted to container, makes sense